src/site/content/en/blog/web-share-target/index.md

```diff
@@ -206,8 +205,8 @@ window.addEventListener('DOMContentLoaded', () => {
 });
 ```
 
-Be sure to use a service worker to precache the `action` page so that it will
-load quickly and work reliably, even if the user is offline.
+Be sure to use a service worker to [precache](https://web.dev/precache-with-workbox/) the `action`
```

Comment:
Someone completely new to this technology might think workbox is the only way to precache. How about linking to an article definition, the appending something like, "an easy way to do this is with workbox."

Reply:
Changed.